docs/architecture/data-pipelines.md

```diff
@@ -0,0 +1,104 @@
+---
+parent: Architecture
+---
+
+# Data Pipelines
+{: .no_toc }
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+- Table of Contents
+{:toc}
+
+
+### Summary
+
+This document outlines how data flows function in OCW-next prior to the rework of the architecture. It serves as a baseline to inform our decision making and scoping of those efforts. Currently some of the pipeline is manual, and some of it is not the most time-efficient.
+
+### Desired Architecture Qualities
+
+- **Decentralized**
+  - Empower course authors to create and publish content in a non-centralized way
+- **Versioned**
+  - Support course authors to be able to make changes without losing historical versions
+  - Support course authors being able to revert to or work off of older versions of course content
+  - Support course authors being able to work on a non-published draft version of a course
+- **Previewable**
+  - Support course authors being able to preview their course material as it will appear when published
+- **Simplicity**
+  - We're aiming to build a favorable user experience for course authors that gets out of their way and makes their jobs as easy as possible
+  - The system should favor simplifying or limiting complex functionality, but design and implement with future flexibility in mind
+
+
+### Current Architecture
+
+| ![Current Architecture Diagram](assets/OCW-next-current.png) |
+|:--:|
+| [full version](assets/OCW-next-current.png) |
```

Comment:
A bit nit-picky, but the current `ocw-next` site is hosted at https://ocw-next.netlify.app.  The proposed first step to host ourselves is to create a Travis build step to push the deployed site into a cloudfronted S3 bucket that will be behind https://ocw-beta.mit.edu.